src/patterns/NPASignupForm.vue

```diff
@@ -104,6 +104,43 @@
 
       <VSpacer size="xl" />
 
+      <Heading
+        v-if="form.plan === 1"
```

Comment:
Would it be possible to put all of this inside a container so that you only have to do `v-if="form.plan === 1"` once?